test/unit/alphabet/detail/alphabet_proxy_test.cpp

```diff
@@ -120,6 +128,10 @@ class alphabet_proxy_example2 : public alphabet_proxy<alphabet_proxy_example2, m
     constexpr alphabet_proxy_example2 & operator=(alphabet_proxy_example2 const &) = default;
     constexpr alphabet_proxy_example2 & operator=(alphabet_proxy_example2 &&) = default;
     ~alphabet_proxy_example2() = default;
+
+    constexpr alphabet_proxy_example2(alphabet_type const a) noexcept
+        : base_t{a}
```

Comment:
Why is this not on one line? I see no need to wrap this? Also when wrapping I think we agreed on putting the operator on the previous line and then continue the remaining expression on the next except for the pipe-notation for the ranges.

Reply:
can you give an example?